appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyEnvironment.java

```diff
@@ -243,19 +248,45 @@ private AttributeMapping(
   /**
    * Helper method to use during the transition from metadata to environment variables.
    *
-   * @param environmentMap the
+   * @param environmentMap the environment
    * @param envKey The name of the environment variable to check first.
    * @param metadataPath The path of the metadata server entry to use as fallback.
    * @param cache The metadata server cache.
    * @return If set the environment variable corresponding to envKey, the metadata entry otherwise.
    */
-  private static String getEnvOrMetadata(Map<String, String> environmentMap, VmMetadataCache cache,
-      String envKey, String metadataPath) {
+  public static String getEnvOrMetadata(Map<String, String> environmentMap, VmMetadataCache cache,
+                                         String envKey, String metadataPath) {
     String envValue = environmentMap.get(envKey);
     return envValue != null ? envValue : cache.getMetadata(metadataPath);
   }
 
   /**
+   * Helper method to get a System Property or Env variable
```

Comment:
'.' and a new line after (to separate description from param).

It would be nice to indicate that "or use the environment variable if not found" (or something like that).